recipe/bld-pkg.bat

```diff
@@ -2,4 +2,6 @@
 
 %PYTHON% %RECIPE_DIR%\builder.py
 
+dir "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\"
```

Comment:
For [CUDA version pre-12]( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/e403f01fd447e48ba3a777a9b642354ab1b7e746/recipe/install_cuda.bat#L15-L27 ), we do install the CUDA Toolkit to [this path]( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/e403f01fd447e48ba3a777a9b642354ab1b7e746/recipe/install_cuda.bat#L217-L218 )

However starting in CUDA 12, the entire CUDA Toolkit is available via Conda packages. So it is not available there

Think we may need to do more debugging on the CMake side to understand where it is looking for the CUDA Toolkit with CUDA 12 so we can understand why it isn't find it